core/conversion/conversion.cpp

```diff
@@ -337,12 +369,28 @@ void ConvertBlockToNetDef(
     } else if (to_eval) {
       auto eval = EvaluateNode(ctx, n);
       if (eval) {
-        if (!eval.value().isTensor()) {
+        if (n->outputs().size() > 1) { // For ListUnpack scenario
+          if (eval.value().isList()) {
+            LOG_DEBUG(
+                ctx->logger,
+                "Found the evaluated value to be a list" << eval.value() << " for node: " << util::node_info(n));
```

Comment:
This is a misleading message i feel as this case is when there are multiple outputs from an evaluator. It makes seem like the object returned is a singular list and not a list of results.